manifest.json

```diff
@@ -24,8 +24,20 @@
             "suggested_key": {
                 "default": "Ctrl+Down"
             }
+        },
+        "toggle-new-tab-option": {
+            "description": "Toggle new background tab option",
```

Comment:
I think `Toggle "Add background tabs after current" option` would be more descriptive.